marlowe-playground-client/src/Marlowe/Linter.purs

```diff
@@ -97,6 +98,7 @@ instance showWarningDetail :: Show WarningDetail where
   show UnreachableCaseEmptyChoice = "This case will never be used, because there are no options to choose"
   show UnreachableCaseFalseNotify = "This case will never be used, because the Observation is always false"
   show UnreachableContract = "This contract is unreachable"
+  show UndefinedChoice = "The contract tries to use a ChoiceId that has not been input by a When"
```

Comment:
Perhaps this message (and `UndefinedUse`) should not use the word `tries` but in stead should explain that the default value of `0` will be used?

Reply:
Good idea, I don't want to wait for the CI again just for that, but I will change it when I get a chance :)